src/components/Form/Form.tsx

```diff
@@ -1,110 +1,161 @@
+            }
 
-    return (
-        <form className={styles.form}>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Title
-                    <input
-                        value={values.title}
-                        onChange={e => setValues('title', e.target.value)}
-                        type='text'
-                        placeholder='Title'
-                        className={styles.input}
-                        required
-                    />
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Release Date
-                    <input
-                        value={values.date}
-                        onChange={e => setValues('date', e.target.value)}
-                        type='date'
-                        placeholder='Select Date'
-                        className={styles.input}
-                        required
-                    />
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Movie URL
-                    <input
-                        value={values.urlMovie}
-                        onChange={e => setValues('urlMovie', e.target.value)}
-                        type='text'
-                        placeholder='Movie URL here'
-                        className={styles.input}
-                        required
-                    />
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Genre
-                    <MultiSelect onChange={values => setValues('genre', values)} initialValues={values.genre}/>
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Overview
-                    <input
-                        value={values.overview}
-                        onChange={e => setValues('overview', e.target.value)}
-                        type='text'
-                        placeholder='Overview here'
-                        className={styles.input}
-                        required
+            if(!values.release_date) {
+                error.release_date = 'Date is required'
+            }
+
+            if(!values.poster_path) {
+                error.poster_path = 'Movie url is required'
+            }
+
+            if(!values.genres?.length) {
+                error.genres = 'Select at least one genre is proceed'
+            }
+
+            if(!values.overview) {
+                error.overview = 'Overview is required'
+            }
+
+            if(!values.runtime) {
+                error.runtime = 'Runtime is required'
+            }
+
+            return error
+        }
```

Comment:
I think it's better to create separate function:
```js
const validate = (values) => {
  return {
    ...(!values.title && { title: "Title is required" }),
    ...(!values.release_date && { release_date: "Date is required" }),
    ...(!values.genres?.length && {
      genres: "Select at least one genre is proceed"
    }),
    ...(!values.overview && { overview: "Overview is required" }),
    ...(!values.runtime && { runtime: "Runtime is required" }),
  };
};
```

Reply:
Thanks, this is fixed